Unnecessary escape character: \/

### Wayfinder Version

0.1.4

### Laravel Version

12.8.1

### PHP Version

8.4.6

### Description

The frontend files generated by Wayfinder have unnecessary escaping which prevents the Github lint.yml quality action from succeeding by throwing errors:

  18:11  error  Unnecessary escape character: \/  no-useless-escape
  18:25  error  Unnecessary escape character: \/  no-useless-escape
  18:31  error  Unnecessary escape character: \/  no-useless-escape

Example:
```ts
edit.definition = {
    methods: ['get','head'],
    url: '\/settings\/password',
}
```

I'm on the React starter kit.

### Steps To Reproduce

1. Install the react starter kit
2. Install wayfinder and generate frontend files
3. Run `npx eslint resources/js/wayfinder/actions/**/*.ts resources/js/wayfinder/routes/**/*.ts --fix`

--- a/resources/method.blade.ts
+++ b/resources/method.blade.ts
@@ -9,7 +9,7 @@
 
 {!! $method !!}.definition = {
     methods: [@foreach ($verbs as $verb)@js($verb->actual){!! when(! $loop->last, ',') !!}@endforeach],
-    url: @js($uri),
+    url: {!! $uri !!},
 }
 
 @include('wayfinder::docblock')

--- a/resources/multi-method.blade.ts
+++ b/resources/multi-method.blade.ts
@@ -9,6 +9,6 @@
 
 {!! when(!$isInvokable, 'export ') !!}const {!! $method !!} = {
 @foreach ($routes as $route)
-    @js($route['uri']): {!! $route['tempMethod'] !!},
+    {!! $route['uri'] !!}: {!! $route['tempMethod'] !!},
 @endforeach
 }{{PHP_EOL}}

--- a/src/Route.php
+++ b/src/Route.php
@@ -7,6 +7,7 @@ use Illuminate\Contracts\Routing\UrlRoutable;
 use Illuminate\Routing\Route as BaseRoute;
 use Illuminate\Routing\RouteAction;
 use Illuminate\Support\Collection;
+use Illuminate\Support\Js;
 use Illuminate\Support\Str;
 use Laravel\SerializableClosure\Support\ReflectionClosure;
 use ReflectionClass;
@@ -94,11 +95,13 @@ class Route
 
         $scheme = $this->scheme() ?? '//';
 
-        return str($this->base->uri)
+        $uri = str($this->base->uri)
             ->start('/')
             ->when($this->domain() !== null, fn ($uri) => $uri->prepend("{$scheme}{$this->domain()}"))
             ->replace($defaultParams->keys()->toArray(), $defaultParams->values()->toArray())
             ->toString();
+
+        return Js::from($uri, JSON_UNESCAPED_SLASHES)->toHtml();
     }
 
     public function scheme(): ?string

